src/shogun/evaluation/SigmoidCalibration.cpp

```diff
@@ -42,28 +42,75 @@
 
 using namespace shogun;
 
-CSigmoidCalibration::CSigmoidCalibration() : CCalibration()
+CSigmoidCalibration::CSigmoidCalibration()
```

Comment:
it should call the ctor of `CCalibration`

Reply:
Yes, my bad